src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsLoggingBuilderExtensions.cs

```diff
@@ -21,7 +21,7 @@ public static class ApplicationInsightsLoggingBuilderExtensions
         public static ILoggingBuilder AddApplicationInsights(
             this ILoggingBuilder builder)
         {
-            return builder.AddApplicationInsights(null);
+            return AddApplicationInsights(builder, null);
```

Comment:
This was an ambiguous reference... I'm assuming the new AspNetCore package includes the `AddApplicationInsights()` extension. Wasn't sure if it was worth updating to that or not...

Reply:
There is no need to update to our new loggerprovider. But please consider marking this method as obsoltete, and provide a new one with AppApplicationInsightsWebJob